exporter/trace/stackdriver/stackdriver.go

```diff
@@ -162,6 +186,71 @@ func (e *Exporter) uploadToStackdriver(spans []*trace.SpanData) {
 	}
 }
 
+// FromRequest extracts a Stakdriver Trace span context from incoming requests.
+func (e *Exporter) FromRequest(req *http.Request) (sc trace.SpanContext, ok bool) {
+	h := req.Header.Get(httpHeader)
+	// See https://cloud.google.com/trace/docs/faq for the header format.
+	// Return if the header is empty or missing, or if the header is unreasonably
+	// large, to avoid making unnecessary copies of a large string.
+	if h == "" || len(h) > 200 {
+		return trace.SpanContext{}, false
+	}
+
+	// Parse the trace id field.
+	slash := strings.Index(h, `/`)
+	if slash == -1 {
+		return trace.SpanContext{}, false
+
```

Comment:
Spurious line, let's remove it.

Reply:
What do you mean by spurious? Are you suggesting to use naked returns?

If so, we end up sometimes using the naked returns and sometimes not. For example, the following case is naked return friendly.

```
 if slash == -1 {
 		return trace.SpanContext{}, false
}
```

And the following is not because ok becomes false. I don't want to assign the right values right before returning.

```
o, err := strconv.ParseUint(h[2:], 10, 64)
 	if err != nil {
 		return trace.SpanContext{}, false
```

In order to be consistent and be more readable, I prefer the current style.

Reply:
By `spurious line` I meant, you have a newline after the return statement and before the closing brace e.g
```go
if foo {
    return bar

}
```